src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -217,38 +217,40 @@ MessagePtr TwitchMessageBuilder::build()
                 // Actually just text
                 QString linkString = this->matchLink(string);
 
-                Link link;
+                Link link = Link();
```

Comment:
Remove `= Link()`.

`Link link;` already calls the default constructor `Link()`.